File: EX5 revisao/repeticao/contador.java
import java.util.Scanner;

public class contador {
    public static void main(String[] args) {
        Scanner gi = new Scanner(System.in);

        System.out.print("Digite o número limite: ");
        int limite = gi.nextInt();

        int n = 0;
        while (n <= limite) {
            System.out.println(n);
            n++;
        }
    }
}

File: EX5 revisao/repeticao/contador.c
#include <stdio.h>

int main() {
  int n,limite;
  n = 0;
  printf("Digite um número a ser contado");
  scanf("%d", &limite);
  printf("%d", n);
  while(n <= limite){
    printf("%d\n", n);
    n++;
    
  }
  return 0;
}